### ISSUE TITLE
Create Book (Preview) generates odd book structure

### ISSUE THREAD

**Issue by MsSQLGirl** [microsoft/azuredatastudio#9780]
- Azure Data Studio Version:

> Version: 1.17.0-insider (user setup)
> Commit: be83b31e378774ed14879fb1889c47422e68bea5
> Date: 2020-03-27T06:21:06.051Z
> VS Code: 1.42.0
> Electron: 7.1.11
> Chrome: 78.0.3904.130
> Node.js: 12.8.1
> V8: 7.8.279.23-electron.0
> OS: Windows_NT x64 10.0.18363

Steps to Reproduce:

1. Setup the directory of the notebook files to be imported (source files)
MyFolder\Notebook01.ipynb
MyFolder\Notebook02.ipynb
MyFolder\Notebook03.ipynb

2. Use Create Book (Preview) and follow the instructions.
3. The TOC created looks like this:
![image](https://user-images.githubusercontent.com/6405919/77837996-83aaed80-7124-11ea-9ee0-eca8681c2962.png)

```
- title: Notebook03
  url: /Notebook03
  not_numbered: true
  expand_sections: true
  sections:  
  - title: Notebook01
    url: /Notebook01

  - title: Notebook02
    url: /Notebook02
```

Expected TOC should look like:
```
- title: Notebook01
  url: Notebook01
- title: Notebook02
  url: Notebook02
- title: Notebook03
  url: Notebook03
```
![image](https://user-images.githubusercontent.com/6405919/77838103-7f330480-7125-11ea-99a1-6402da40ce23.png)

**Comment by MaddyDev** [comment]
@MsSQLGirl  I've created it that way, Usually every books has a readme markdown (check the TSG book from Staurt). So I added a title section for every book. When there is no markdown, it takes the last file and treats as the title file. 
If having a title thing isn't common for Jupyter Book, I'll modify it accordingly. Please let me know if that's a common practice.
@chlafreniere  Would like you insights as well.

**Comment by MaddyDev** [comment]
fixed #9825
